src/components/test/common-tests.js

```diff
@@ -151,34 +151,52 @@ export function testPresentationalComponent(
 /**
  * Set of tests common to all base components
  *
- * @param {FunctionComponent} Component
+ * @param {import('preact').FunctionComponent} Component
+ * @param {CommonTestOpts} opts
  */
-export function testBaseComponent(Component) {
-  describe('Common base component functionality', () => {
+export function testBaseComponent(
+  Component,
+  { componentName, createContent = createComponent, elementSelector } = {}
+) {
+  const displayName = componentName ?? Component.displayName ?? Component.name;
+  describe(`Common base presentational functionality for ${displayName}`, () => {
```

Comment:
```suggestion
  describe(`Common base functionality for ${displayName}`, () => {
```

"presentational" seems superfluous to me.